robot/camera.py

```diff
@@ -56,44 +54,20 @@ def see(self):
                     raise
 
 
-class ResultList(MutableSequence):
+class ResultList(list):
```

Comment:
Arguably OOS of this PR, but it might be nice to make this immutable, by overriding `__setitem__` to raise an exception, or extending `Sequence`

Reply:
I'm not sure whether we need to do that (though I don't feel strongly). Agree it's out of scope for this PR.